### ISSUE TITLE
Memory leak: event listeners aren't removed (?)

### ISSUE THREAD

**Issue by johanrd** [hokulea/aria-voyager#175]
1) Go back and forth to a route with an element using the menu modifier, and take a heap snapshot between each time.
2) See that for every time, a new instance of Menu and MenuNavgation is retained (accumulating) in memory 

<img width="691" alt="Screenshot 2024-10-13 at 23 43 14" src="https://github.com/user-attachments/assets/f819f453-7f4e-4206-be45-41038997b820">

Most probable cause seems to be that added event listeners in [control](https://github.com/hokulea/aria-voyager/blob/a8223b55fec5862bc334855819c2bdfeea97a1ee/packages/aria-voyager/src/controls/control.ts#L123) aren't removed on component teardown.

**Comment by gossi** [comment]
Good catch. I was hoping to remove all references so JS GC can do its job. Apparently it's not the case.

Thanks for doing the investigation 👍

**Comment by gossi** [comment]
I had some dispose functionality already lying around. I did a quite _cheap_ implementation in #269 - I didn't profile though. It is more to get it chained up, though it may have an effect already.

Released in `0.1.0`
